index.html

```diff
@@ -1238,6 +1238,369 @@ <h2>Evidence</h2>
+        <figcaption style="text-align: center;">
+Subject Holder Relationships in Verifiable Credentials.
+        </figcaption>
+      </figure>
+
+     <p>
+The following sections describe how each of these instances are handled in 
+the data model
+     </p>
+
+
+    <section>
+        <h2>Credential Uniquely Identifies Subject</h2>
+
+     <p>
+In this case, the claim may contain multiple properties that each provide 
+an aspect of the identity of the subject, and which together unambiguously identify
+the subject. In this case, it does not matter who the holder is, since the verifier
+can unambiguously identify the subject of the verifiable credential.
+   </p>
+
+        <pre class="example nohighlight" title="An example of a credential 
+that uniquely identifies a subject">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "ProofOfAgeCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01",
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"name": "Mr John Doe",
+		"address": "10 Some Street, Anytown, ThisLocal, Country X",
+		"dateOfBirth": "2001-01-01",
+		"gender": "male",
+		"placeOfBirth": "Anytown, ThisLocal, Country X",
+		"nationality": "X",
+	},
+	"proof": { ...
+	}
+}
+        </pre>
+
+
+   </section>
+
+    <section>
+       <h2>Subject Delegates to Holder</h2>
+
+     <p>
+This is supported in the data model in two different ways: through the use of either recursion or verifiable presentations. Both methods involve the subject issuing a new verifiable credential to the delegated holder.
+    </p>
+
+    <p>
+This process of recursion could continue without restriction. Therefore 
+a new property “delegatable”, is added to the data model.
+
+     </p>
+
+   <section>
+
+   <h3> Delegatable Property </h3>
+
+  <p>
+The delegatable property, a Boolean, states whether a credential is 
+delegatable or not. If it is present with the value true, or not present, 
+then the credential is delegatable to a third party. If it is present with 
+the value false then it is not delegatable. A verifier that receives a verifiable 
+credential that is not delegatable MUST reject any verifiable credential that has
+been delegated from it.
+
+  </p>
+
+        <pre class="example nohighlight" title="An example of a credential that is not 
+delegatable">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "ProofOfAgeCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01",
+	"delegatable": false,
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"ageOver": 21,
+	},
+	"proof": { ...
+	}
+}
+        </pre>
+
+       </section>
+       
+       <section>
+
+     <h3> Delegation through Recursion </h3>
+
+    <p>
+With recursion, the subject issues a credential to the delegated holder in which: 
```

Comment:
This creates confused deputy attacks, doesn't it? I'm struggling to understand what the use case is here other than ones we know we don't want "Here, use my access badge"?

Reply:
No I do not believe that delegation of an organisational role from one person to another person creates a confused deputy, as the delegate inherits the permissions of the delegator, which is what is intended.
Could you please explain why you think the delegate might be a confused deputy?

Reply:
It does create a confused deputy problem because you're authorizing based on identity, and if you're authorizing based on identity you're holding onto a larger set of credentials about "who you are" rather than "what you're doing".  It's not that the inheriting of permissions is *unintended* in the confused deputy... a confused deputy is usually behaving as designed, and that is part of the point of the confused deputy problem.  I encourage reading the [original confused deputy example](http://www.cap-lore.com/CapTheory/ConfusedDeputy.html).  The problem is that the program holding that authority cannot distinguish the *purpose* for which it should be using its wide range of authority.  This is why an identity-centric approach to authority will always lead to confused deputies... not because they are misprogrammed, but because the problem is inherent in that design.

Reply:
@cwebber I beg to disagree with you. If I intend someone to take over my role, say because I am on holiday or ill for a period of time, then I do want them to inherit ALL my privileges. The purpose is clearly to inherit all my authority, otherwise they could not take on my role in its entirety. Most organisations today operated in this way. They cannot function without delegation.

Reply:
@David-Chadwick, I think there are three statements you're making here that we need to separate:

1. Organizations need to delegate authority.

I think everyone agrees on that.

2. The way organizations delegate authority are Verifiable Credentials.

We do not have consensus on that and I really doubt we'll get there because this creates confused deputy and ambient authority issues. We have at least @cwebber, @dlongley, and myself in agreement on that (having put a tremendous amount of thought into this).

3. You want to be able to grant ALL of your privileges to an entity.

This is the definition of confused deputy AND ambient authority and the academic and corporate literature on why this is a terrible security design paradigm goes back to the 1970s.

At this point, I suggest that we wipe the section you've authored on delegation, put an issue marker in there pointing to an issue on delegation when subject != holder, and sort the solution out in that issue.

I don't want to delay getting this PR merged due to this one issue, which doesn't seem like it's going to be resolved any time soon.

Reply:
I want to clarify -- the ability to granting all privileges to another entity is not the confused deputy problem/ambient authority, rather, a system where there is only a binary choice of delegating all privileges (*for all targets/verifiers*) or no delegation at all leads straight to it.

People should be able to delegate authority. The system that enables it should allow for granularity from zero delegation to full (and everything in between). Furthermore, authority should be *targeted*.

The VC model is about identity not authority. It is also designed, primarily, to address use cases where a VC can be issued by one party and verified *by many different parties*. There is not a single target. That is not an appropriate design for decentralized authorization and delegation. But you can build something appropriate on top of it -- that uses identity as a starting place to instantiate rights/privileges. This is what OCAP-LD is for.

Reply:
@dlongley. Thankyou for your clarification of confused deputy, which is very helpful to this discussion. I will go further than your clarification however, and say that confused deputy is not in essence to do with delegating authority, nor whether there is a binary choice in the delegation, but rather, it is when one party asks another party (the deputy) to perform an action that the deputy is authorised to perform but the requestor is not. In RBAC, the process of role delegation is separate from the process of requesting an action, so providing the process for requesting an action ensures that the requestor has full authority for the requested action, then the confused deputy problem cannot happen. Furthermore hierarchical RBAC allows for fine grained delegation of authority by delegating a subordinate role to the one possessed, so your issue of binary choice is not relevant here. Since the text on delegation is only concerned with delegation and not with requesting actions, then I do not see that the confused deputy problem is relevant to the proposed text.